examples/simple-monorepo/README.md

```diff
@@ -1,6 +1,45 @@
-# ICMK Simple Example
+# IZE Simple Example
 
- ```shell script
-make infra
-make tunnel
+### Commands for generate terraform files
+
+```shell
+ize -c ./examples/simple-monorepo/.infra/env/testnut/ize.toml env terraform
+```
+
+### Commands for deploy infrastructure
+
+```shell
+ize -c .infra/env/testnut/ize.toml deploy infra
+ize -c .infra/env/testnut/ize.toml destroy infra
+```
+
+### Set config path via ENV
+```shell
+export IZE_CONFIG_FILE=<path to your infra folder>/.infra/env/dev/ize.toml 
+```
+
+### Establish SSM tunnel
+```shell
+ize tunnel ssh-key
+ize tunnel up
 ```
+
+### Upload secrets
+```shell
+ize secret set --file .infra/env/testnut/secrets/example-service.json --type ssm
+```
+
+### Remove secrets
+```shell
+ize secret remove --type ssm --path /testnut/example-service
```

Comment:
TODO (not in this PR): I think we should think of switchig from `--type` and `--path` to `ssm://testnut/example-service` approach. WDYT?

Reply:
I think this is a good idea, but its
need to be discussed with everyone